src/fragments/lib/storage/js/download.mdx

```diff
@@ -9,17 +9,20 @@ await Storage.get(key: string, config: {
   download?: boolean, // defaults to false
   expires?: number, // validity of the URL, in seconds. defaults to 900 (15 minutes)
   contentType?: string // set return content type, eg "text/html"
+  validateObjectExistence?: boolean // defaults to false
 });
 ```
 
-`Storage.get` returns a signed URL `string` to your file, if `download` is false, which is the default. You can use this to create a download link for people to click on. This is our recommended option. Note that this method **does not check if the file actually exists** as that would involve an extra API call.
+`Storage.get` returns a signed URL `string` to your file, if `download` is false, which is the default. You can use this to create a download link for people to click on. This is our recommended option. Note that this method **does not check if the file actually exists by default**. But you can check for the existence of file by setting `validateObjectExistence` to `true`.
```

Comment:
why is it our recommended option?

Reply:
This is a good question. If we have set download to true it would return complete Blob object in the API call response. In order to avoid loading complete object with get call we recommend setting download flag to false. please let me know if have any concerns.

Reply:
I think we need to add a bit of a description on why that is our recommended option. what do you think @jimblanc and @stocaaro ?